src/common/pages/test/icon-button-page/IconButtonPage.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import ArrowRightRoundedIcon from "@mui/icons-material/ArrowRightRounded";
+import { Box } from "@mui/material";
+
+import IconButton from "@/common/components/icon-button";
+
+import { wrapper } from "./IconButtonPage.style";
+
+const IconButtonPage: React.FC = () => {
+  return (
+    <Box sx={wrapper}>
+      <IconButton
+        icon={<ArrowRightRoundedIcon />}
+        size="small"
+        isDisabled={true}
+        reactionOnClick={false}
```

Comment:
You don't need to write 'true' / 'false' when you create a component. You can just use
      <IconButton
         icon={<ArrowRightRoundedIcon />}
         size="medium"
         isDisabled
       />
       
       When you put only prop's name, it means 'true'. 
       Example:
       isDisabled = {true} is the same like isDisabled